pycqed/measurement/measurement_control.py

```diff
@@ -107,6 +101,32 @@ def __init__(self, name: str,
     # Functions used to control the measurements #
     ##############################################
 
+    def open_plotting_windows(self):
```

Comment:
I am generally paranoid about any changes to MC but I guess this looks good :) (especially since it is owrking on your system).